lib/Optimizer/Lower.cpp

```diff
@@ -187,6 +187,25 @@ void lowerReluNode(Function *F, ReluNode &R) {
   R.getResult().replaceAllUsesOfWith(relu);
 }
 
+/// Lower a quantized Log by creating an IntLookupTable with a new mapping given
+/// the input and output quantization parameters.
+void lowerQuantizedLogNode(Function *F, LogNode *LN) {
```

Comment:
nit: static func and const LogNode *

Reply:
I was matching the rest of them in here but I agree it should be the case. I will do a cleanup followup PR here to fix them all at once.